response.go

```diff
@@ -11,6 +11,21 @@ func Arguments() []string {
 	return os.Args[1:]
 }
 
+// NormalizedArguments re-normalizes the user arguments provided via Alfred.
+// This isn't necessary for every workflow, specifically only when you're working with special characters.
+// For more info on this topic, please refer to this thread: http://www.alfredforum.com/topic/2015-encoding-issue/
+func NormalizedArguments() (normalizedArgs []string, err error) {
+	for _, e := range Arguments() {
+		var normalizedElement string
+		normalizedElement, err = Normalize(e)
+		if err != nil {
+			return
```

Comment:
I would've expected to check the error on the other side and if present, ignore the partially normalized input altogether. But you're right, people can also choose to ignore the error and will get stumped as to why they're only getting a part of the results.